lib/fastly/service.rb

```diff
@@ -1,8 +1,10 @@
+# frozen_string_literal: true
+
 # Service object
 class Fastly
   # Represents something you want to serve - this can be, for example, a whole web site, a Wordpress site, or just your image servers
   class Service < Base
-    attr_accessor :id, :customer_id, :name, :comment, :versions
+    attr_accessor :id, :customer_id, :name, :comment
```

Comment:
This one was kinda weird. The cop was `Method Fastly::Service#versions is defined at both lib/fastly/service.rb:7 and lib/fastly/service.rb:75.`. Which if you check below, we have a method called `versions`, but it's also treated as a reader on this line. 🤷

Reply:
I think it's because `attr_accessor` adds both the getter and setter, and we redefine the getter below. @andriantoniades has a PR open to address this also (a warning is generated), so that's interesting we got a cop warning and a ruby warning.